Nighttime-Operations/Observatory/DIMM/ops.rst

```diff
@@ -0,0 +1,194 @@
+.. |author| replace:: *isotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _DIMM-Ops:
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. Limited support from the user is expected. 
+
+The vendor provided DIMM software will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+At dawn, the telescope will park itself, and the dome will close. 
+This cycle repeats the following evening, unless :ref:`manually shutdown <Dimm-Shutdown>`, when the sun sets.
+
+The `ameba` program is continously checking that the environment conditions are stable and clear, based on the data delivered by `tt-meteo`.
+In case of adverse weather, `ameba` will close the dome until conditions clear. 
+
+Currently, there is no incoming Cerro Pachón weather information feeding the `tt-meteo` program. 
+
+Thus prior to starting up the DIMM, there are additional steps outlined in the :ref:`prerequisites section <Dimm_StartUp-Prerequisites>`.
+
+.. _Dimm_StartUp-Prerequisites: 
+
+Prerequisites: Weather Data
+===============================
+
+Every time the DIMM is restarted, the weather startup values are set so the operations are disabled for safety reasons. 
+
+To allow operations, mock weather data into the `tt-weather` service must be entered in the following manner:
+
+.. important::
+    Ensure the weather conditions are stable and safe before attempting the DIMM startup. 
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
```

Comment:
Consider:

When connected to the LSST WAP network, open a terminal and connect to the DIMM VM using:

Reply:
👍